src/main/java/org/gradle/profiler/ScenarioInvoker.java

```diff
@@ -44,23 +44,23 @@ protected R runMeasured(BuildContext buildContext, BuildMutator buildMutator, Bu
     /**
      * Returns a {@link Supplier} that returns the result of the given command.
      */
-    BuildStepAction<BuildInvocationResult> measureCommandLineExecution(List<String> commandLine, File workingDir, File buildLog) {
-        return new BuildStepAction<BuildInvocationResult>() {
+    BuildStepAction<R> measureCommandLineExecution(List<String> commandLine, File workingDir, File buildLog) {
```

Comment:
Should this method go to `BuildToolCommandLineInvoker` as well?

Reply:
Done.